ddl/ddl_worker_test.go

```diff
@@ -710,3 +711,23 @@ func (s *testDDLSuite) TestParallelDDL(c *C) {
 	tc = &TestDDLCallback{}
 	d.SetHook(tc)
 }
+
+func (s *testDDLSuite) TestDDLPackageExecuteSQL(c *C) {
+	store := testCreateStore(c, "test_run_sql")
+	defer store.Close()
+
+	RunWorker = true
+	d := testNewDDL(context.Background(), nil, store, nil, nil, testLease)
+	testCheckOwner(c, d, true)
+	defer d.Stop()
+	worker := d.generalWorker()
+	c.Assert(worker, NotNil)
+
+	// In test environment, worker.ctxPool will be nil, and get will return mock.Context.
+	// We just test that can use it to call sqlexec.SQLExecutor.Execute.
+	sess, err := worker.sessPool.get()
+	c.Assert(err, IsNil)
+	defer worker.sessPool.put(sess)
+	se := sess.(sqlexec.SQLExecutor)
+	_, _ = se.Execute(context.Background(), "create table t(a int);")
```

Comment:
Do we need to check the error？

Reply:
No need, please see the comments in line 726